core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -1285,7 +1285,7 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     var electResult = client.electLeaders(ElectionType.PREFERRED, Set(partition1).asJava)
     var exception = electResult.partitions.get.get(partition1).get
     assertEquals(classOf[ElectionNotNeededException], exception.getClass)
-    assertEquals("Leader election not needed for topic partition", exception.getMessage)
+    assertEquals(org.apache.kafka.common.protocol.Errors.ELECTION_NOT_NEEDED.message(), exception.getMessage)
```

Comment:
Is the assertion useful in the first place? Verifying the exception type seems good enough

Reply:
True, we use `ApiError` for this, which pulls in the right message programmatically. Removed.